src/PerfView/PerfViewData.cs

```diff
@@ -6860,6 +6869,83 @@ string GetAllocationType(CallStackIndex csi)
         }
 
         #region private
+
+        /// <summary>
+        /// Checks if RuntimeStart events exist for the selected processes and warns if missing.
+        /// This indicates whether the processes were started after tracing began, which is required
+        /// for proper type information in unsampled allocation traces.
+        /// </summary>
+        private void WarnAboutMissingTypeInfoForProcesses(Window parentWindow, List<int> processIDs, string sourceName)
+        {
+            bool showWarning = true;
+
+            // Only check for allocation-related views
+            if (!sourceName.Equals("GC Heap Alloc Ignore Free") &&
```

Comment:
This hardcoding is pretty rough.  There are a lot of individual strings of things like "GC Heap Net Mem" in the codebase.  It would be nice if these were some kind of `SourceKind` enum that translated to this string instead of hardcoding everywhere, but that's a change or topic for another day.